src/portal/src/app/project/tag-retention/tag-retention.component.ts

```diff
@@ -167,7 +169,34 @@ export class TagRetentionComponent implements OnInit {
         this.addRuleComponent.isAdd = false;
         this.ruleIndex = -1;
     }
-
+    disable(index) {
```

Comment:
this disable method and enable method can be refactor to one method with different parameters.

Reply:
fixed